models/src/main/scala/coop/rchain/models/rholang/implicits.scala

```diff
@@ -436,7 +436,7 @@ object implicits {
         case ETupleBody(e)                                => e.locallyFree
         case ESetBody(e)                                  => e.locallyFree.value
         case EMapBody(e)                                  => e.locallyFree.value
-        case EVarBody(EVar(v))                            => VarLocallyFree.locallyFree(v, depth)
+        case EVarBody(EVar(v))                            => VarInstanceLocallyFree.locallyFree(v.varInstance, depth)
```

Comment:
We can remove all `depth` from this file, e.g. here or from `prepend` methods. But this will require a lot of changes in the Reducer, so I can do it while working on new types.